web/policies/stripe_account_policy.ex

```diff
@@ -0,0 +1,14 @@
+defmodule CodeCorps.StripeAccountPolicy do
+  import CodeCorps.Helpers.Policy,
+    only: [get_membership: 2, get_role: 1, admin_or_higher?: 1, owner?: 1]
+
+  alias CodeCorps.StripeAccount
+  alias CodeCorps.User
+  alias Ecto.Changeset
+
+  def index?(%User{admin: true}, %StripeAccount{}), do: true
+  def index?(%User{} = user, %StripeAccount{} = stripe_account), do: stripe_account |> get_membership(user) |> get_role |> admin_or_higher?
```

Comment:
I'll contact Alan, but it seems like the convention here is to use a scope.

Bodyguard offers [scopes](https://github.com/schrockwell/bodyguard#policy-scopes), while, sadly, our current canary offers no such thing.

Since we're using `ja_resource`, perhaps a `def records, do: records_current_user_is_authorized_for` would be the correct approach? If we ever switch to bodyguard, maybe later on it would make sense to move that code into a `scope`.

Looking at the API blueprint, it doesn't look an `organization_id` parameter is required. Should we make it a requirement?

We would probably have something like a records callback matching a request without an org id, which returns a zero records, as well as one with an org id, which returns zero records if current user is not org owner, and all records for current org if user is org owner. At that point, we can have an `index?` that accepts a current user and an organization or keyword list with an organization id.

Really, even without contacting Alan, it looks to me like scopes would be the way to go and this is as close as we can get to a scope, without switching from canary to a bodyguard. Really, it's a scope in anything other than the name.